src/components/Onboarding/onboarding.scss

```diff
@@ -1,35 +1,39 @@
 .analysis-card {
-    background-color: hsla(248, 21%, 20%, 0.94) !important;
-    border-radius: 34px !important;
-    height: 100%;
-    min-height: 550px;
-    padding: 24px;
-  }
+  background-color: hsla(248, 21%, 20%, 0.94) !important;
+  border-radius: 34px !important;
```

Comment:
Technically, the `rounded-lg` class can be added to the actual image, but I guess no big deal

Reply:
this one I actually didn't add despite whatever it's saying